locales/en/app.ftl

```diff
@@ -828,4 +828,7 @@ email-addresses-title = Email Addresses
 # This is a standardized breach overview blurb that appears on all breach detail pages.
 # $breachTitle is the name of the breached company or website.
 # $breachDate and $addedDate are calendar dates.
-On { $breachDate }, { $breachTitle } suffered a breach. Once the breach was discovered and verified it was added to our database on { $addedDate }.
+breach-overview = On { $breachDate }, { $breachTitle } suffered a breach. Once the breach was discovered and verified it was added to our database on { $addedDate }.
+
+# Title appearing on the Preferences dashboard. 
+monitor-preferences = {-brand-monitor} Preferences
```

Comment:
We have `-product-name`, which is "Firefox Monitor". Can we use that?

Reply:
Also, is it OK to give up on the bold for Preferences?

Reply:
Yes, it would have needed to go either way.

Reply:
To be clear: either we add another term for *Monitor* (and it would be `-product-short-name`), or we update the string. `-brand-monitor` doesn't exist.

Reply:
The headline is "Monitor Preferences", so I think it will require a new string for { -brand-Monitor }